crates/bevy_render/src/camera/projection.rs

```diff
@@ -112,3 +115,25 @@ impl Default for OrthographicProjection {
         }
     }
 }
+
+/// Given coordinates in world space, use the camera and window information to compute the
+/// screen space coordinates.
+pub fn world_to_screen(
+    world_space_coords: &GlobalTransform,
+    camera: (&Camera, &GlobalTransform),
+    window_resource: &Res<Windows>,
```

Comment:
I want to avoid accepting Res/ResMut in functions like this. As a general rule of thumb, we should try to avoid using ECS types where they aren't needed. It decreases complexity and increases re-usability. This kind of relates to the previous comment about "data origin".